schemas/Clinical/index.js

```diff
@@ -387,7 +387,7 @@ const convertClinicalSubmissionDataErrorToGql = errorData => {
     donorId: get(errorData, 'info.donorSubmitterId', ''),
 
     // errorData.info.value may come back as null if not provided in uploaded file
-    value: get(errorData, 'info.value', ''),
+    value: get(errorData, 'info.value', '') || '',
```

Comment:
do we expect the same for `donorId`?

Reply:
that shouldn't be possible because every record has a submitterDonorId, clinical wouldn't have accepted the file if it was missing. But I guess it doesn't hurt to add that check.

Reply:
the gql schema defines the field as a `String!`, so regardless of what the possibility, we have to ensure a string comes through to it. If such guarantee is not the case, it should be typed as `String` :)

Reply:
agreed